Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp

```diff
@@ -585,9 +590,25 @@ namespace AZ
                 AZ_Warning(FeatureProcessorName, false, "Sampling count exceed the limit.");
                 count = Shadow::MaxPcfSamplingCount;
             }
+
+            //Remap the count value to an enum value associated with that count.
+            ShadowFilterSampleCount samplingCountMode = ShadowFilterSampleCount::Tap0;
```

Comment:
just making sure, _samplingCountMode_ will never be _ShadowFilterSampleCount::Tap0_?

Reply:
yeah. Not at the moment. We could in theory add support for that mode in future for low end devices.